### DISCUSSION TITLE
Garmin 1030 & RCT715 Turn On when radar activated

### DISCUSSION THREAD

**Discussion by rbennu** [maca88/SmartBikeLights#48] [General]
Trying to configure so that the light comes on in Peloton mode when the radar is activated.  Currently the light stays on.  Current configuration  ###0,73535488::1#2,2!GPS:1:5:0:0I]-1]1!:1:5:0:0D=1##6,5:Varia 715:0:16777215!2,:-1,Off:0!1,Solid:4!1,Peloton:5!1,Day Flash:7!1,Night Flash:6#0::#0:0#123!:123!#0#0#B2713##11#1#0

**Comment by maca88** [comment]
You need to lower the Threat operator to "Greater than None" as otherwise it will be activated only for fast vehicles (when you see the red color in Edge). Here is the updated configuration that will work for every approaching vehicle not just the fast ones (orange color):

###0,73535488::1#2,2!GPS:1:5:0:0I]-1]0!:1:5:0:0D=1##6,5:Varia 715:0:16777215!2,:-1,Off:0!1,Solid:4!1,Peloton:5!1,Day Flash:7!1,Night Flash:6#0::#0:0#123!:123!#0#0#B2713##11#1#0

**Comment by rbennu** [comment]
Thank you, I started that way but got confused with a couple things.  After
I load the configuration, the light stays on.  I assumed it would turn off,
is that correct?  I haven't had it on the bike as yet.  When it's sitting
on my desk the light will eventually turn off at the same time the light
network/sensor will become disconnected from the 1030 (showing error 1).
Is this because it's not moving?

**Reply by maca88** [reply]
Oh, I overlooked that your configuration has the "Default mode" set to "Peloton". That is why the light starts with "Peloton" mode. If you want to start with Off, just set the default mode to "Off". Here is the updated configuration:

###0,73535488::1#2,2!GPS:1:5:0:0I]-1]0!:1:0:0:0D=1##6,5:Varia 715:0:16777215!2,:-1,Off:0!1,Solid:4!1,Peloton:5!1,Day Flash:7!1,Night Flash:6#0::#0:0#123!:123!#0#0#B2713##11#1#0

**Comment by rbennu** [comment]
Got it, thank you!
